service/src/main/kotlin/fi/espoo/evaka/placement/PlacementPlanService.kt

```diff
@@ -47,7 +47,8 @@ class PlacementPlanService(
 
         when (application.type) {
             ApplicationType.PRESCHOOL -> {
-                val term = derivePreschoolTerm(startDate)
+                val term = tx.getActivePreschoolTermAt(startDate)?.extendedTerm
```

Comment:
This should be either `finnishPreschool` or `swedishPreschool` depending on the unit instead of `extendedTerm`. The `extendedTerm` should be used when determining the `preschoolDaycarePeriod` below.

Reply:
At this point of the flow we don't yet have only one unit and hence the dates are like best guesses, aren't they? Maybe the `finnishPreschool` would be much better than `extendedTerm` though.

Reply:
Oh yeah, true. In either case, `extendedTerm` is incorrect. I guess `finnishPreschool` is a good default until Svebi decides otherwise 😄